### PR TITLE
added CRUD  controller functions for html and json

### PR DESCRIPTION
**Pull request by SarahPAms**
issue #71

### CODE REVIEW

**Comment by SarahPAms** [comment]
to be used with the controller setup (clearly not properly indented):

Rails.application.routes.draw do



  root to: "pages#home"

  get "attend", to: "pages#attend"
  get "incompany", to: "pages#incompany"
  get "sponsor", to: "pages#sponsor"

  devise_for :admins

  scope '/admin' do
    resources :workshops, :sponsors, :events
    resources :incompanyrequests, only: [:index, :show, :edit, :update, :destroy]
    end
  end

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -1,8 +1,61 @@
 class EventsController < ApplicationController
+  before_action :set_event, only: [:show, :edit, :update, :destroy]
```
Comment (arnoFleming) [inline app/controllers/events_controller.rb]: ty

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -1,8 +1,61 @@
...
+  def edit
+  end
+
+  def create
+    @event = Event.new(event_params)
+
+    respond_to do |format|
+      if @event.save
+        format.html { redirect_to events_path, notice: "Event was successfully created" }
+        format.json { render :show, status: :created, location: @event }
```
Comment (arnoFleming) [inline app/controllers/events_controller.rb]: we don't API today

Comment (arnoFleming) [inline app/controllers/events_controller.rb]: you can leave it, but don't spend time on it now

File: config/routes.rb [hunk 3]
```diff
@@ -10,9 +10,8 @@
 
   devise_for :admins
 
-  resources :workshops
-  resources :sponsors
-  resources :events do
-    resources :attendees
+  namespace :admin do
+    resources :workshops, :sponsors, :events
+    resources :incompanyrequests, only: [:index, :show, :edit, :update, :destroy]
```
Comment (arnoFleming) [inline config/routes.rb]: I think this should be an `resource :in_company_request` - check @Dvessen83

**Pull request merged**